tn_api_v2_core.yml

```diff
@@ -0,0 +1,956 @@
+                    example: "WS"
+                intermediate_stops:
+                    type: array
+                    description: Direct flight between two points with no change in flight number
+                    items:
+                        $ref: '#/components/schemas/DatasourceLegDetails'
+                price:
+                    type: number
+                    description: Price of the segment (fare and tax included).
+                    example: 123.45
+                e_ticketability:
+                    type: boolean
+                    description: E-tickable tickets
+                    example: true
+                cancel_penalty:
+                    type: object
+                    description: Information about cancellation
+                    properties:
+                        amount:
+                            type: number
+                            description: Cancellation penalty amount
+                            example: 0
+                        percentage:
+                            type: number
+                            description: Cancellation penalty percentage
+                            example: 100
+                change_penalty:
+                    type: object
+                    description: Information about changing.
+                    properties:
+                        amount:
+                            type: float
+                            description: Change penalty amount
+                            example: 78
+                        percentage:
+                            type: float
+                            description: Change penalty percentage
+                            example: 0
+
+        MinimalDatasourceLegDetails:
+            description: A list of legs for this segment
+            type: object
+            required:
+                - departure_time
+                - arrival_time
+                - flight_number
+                - plating_carrier
+                - transportation_type
+            properties:
+                departure_time:
+                    type: string
+                    description: Departure time. Example = "2023-03-15T08:00:00"
+                    example: "2023-03-15T08:00:00"
+                arrival_time:
+                    type: string
+                    description: Arrival time. Example = "2023-03-15T10:29:00"
+                    example: "2023-03-15T10:29:00"
+                flight_number:
+                    type: string
+                    description: Flight number. Example = "391"
+                    example: "391"
+                operating_carrier:
+                    type: string
+                    description: Operating carrier. Example = "WS"
+                    example: "WS"
+                transportation_type:
+                    type: string
+                    description: "Transportation vehicle used for this segment."
+                    example: "flight"
```

Comment:
Not a big thing, but we usually don't include the examples in the description so wanted to check that is intentional.

Reply:
kk